migrations/20220825210520_265_users__uin_unique_per_institution.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users ADD CONSTRAINT users_uin_institution_id_key UNIQUE (uin, institution_id);
```

Comment:
Could we use this syntax so we can include `IF NOT EXISTS` (for general robustness): https://github.com/PrairieLearn/PrairieLearn/blob/df2771dd8c10ff21dbe8ba1951468af4e74688f4/migrations/20200913144809_216_questions__course_id__uuid__create_index.sql#L1

Reply:
Yes!

Reply:
Done in d946fa0.